changelog.d/6398.bugfix

```diff
@@ -0,0 +1 @@
+Add validation of format for 3pid.
```

Comment:
Contrary to how towncrier is normally used, we've been putting in newsfragments just for the pull request number, so this file can be removed.

Reply:
Do you will remove it when merging the PR? When I remove the file, all CI tests will fail.

Reply:
Please delete this file.